lib/elixir/lib/module/types/of.ex

```diff
@@ -114,9 +114,7 @@ defmodule Module.Types.Of do
       # will already have checked the type, so we skip the check here
       # as an optimization.
       # TODO: properly handle dynamic. Do we need materialization?
```

Comment:
Does this TODO still apply?

Reply:
No, feel free to yank it and push :)

Reply:
✅ https://github.com/elixir-lang/elixir/commit/cbe0132eaedab29c74462258f36c827e8e6bf06b